.github/workflows/ci.yml

```diff
@@ -1,28 +1,31 @@
 name: CI
-
 on:
   push:
-    branches: [ main ]
+    branches: [main]
   pull_request:
-    branches: [ main ]
+    branches: [main]
 
 jobs:
-  build:
-
+  test:
     runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Referencing the comment I just made on a different PR 🙂 https://github.com/primer/eslint-plugin-primer-react/pull/88#discussion_r1356408977

Reply:
Done! ✨